### PR TITLE
roll 2.1.1

### PR DESCRIPTION
**Pull request by equal-l2**

### CODE REVIEW

File: roll.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 class Roll < Formula
   desc "CLI program for rolling a dice sequence"
-  homepage "http://matteocorti.ch/software/roll.html"
-  url "http://matteocorti.ch/software/roll/roll-2.0.2.tar.gz"
-  sha256 "15b3fb5fe43c61799da3e15574aec4fb414e30e7bfda3b1e4a3b4b398a3a99a9"
+  homepage "http://matteocorti.github.io/roll/"
```
Comment (dunn) [inline roll.rb]: Just a minor thing:
```
homebrew/games/roll:
  * Please use https:// for http://matteocorti.github.io/roll/

Comment (equal-l2) [inline roll.rb]: Fixed. Thanks.

**Comment by dunn** [comment]
@BrewTestBot test this please

**Comment by dunn** [comment]
Sorry for the delay, our precompiled bottles are being uncooperative.

**Comment by mistydemeo** [comment]
@BrewTestBot test this please

**Comment by mistydemeo** [comment]
Thanks!

**Pull request closed without merging**